automated-tests/src/main/java/com/hp/ov/sdk/network/FcoeNetworkResource.java

```diff
@@ -97,4 +104,27 @@ public FcoeNetwork builderUpdate(FcoeNetwork fcoeNetwork) {
         fcoeNetwork.setName(resourceProperties.get("name"));
         return fcoeNetwork;
     }
+
+	public String patchFcoeNetwork(String id) {
+		FcoeNetwork fcoeNetwork = new FcoeNetwork();
+		fcoeNetwork.setName(resourceProperties.get("name"));
+
+        Patch patch = new Patch();
+
+        patch.setOp(Patch.PatchOperation.valueOf(resourceProperties.get("op")));
+        patch.setPath(resourceProperties.get("path"));
+        List<String> scopeUris = fcoeNetwork.getScopeUris();
+        scopeUris.add(SCOPES_URI + "/" + getScopeId(resourceProperties.get("value")));
+        patch.setValue(scopeUris);
+
+        return objToString(client.patch(id, patch));
+	}
+
+    private String getScopeId(String scopeName) {
+        ScopeClient scopeClient = oneViewClient.scope();
+
+        Scope scope = scopeClient.getByName(scopeName).get(0);

```

Comment:
Please add a verification if scope exists, to avoid a NPE.

Reply:
Done